src/main.cpp

```diff
@@ -45,7 +45,8 @@ int main(int argc, char** argv) {
     int elastic_batch_size = 5000;
     int elastic_issue_time = 5000;
 
-    std::string elastic_provenance_index = "provenance";
+    std::string elastic_file_provenance_index = "";
```

Comment:
what will be the file provenance index, was it per project? is this parameter even needed?

Reply:
fixed - removed the index